File: docs/sdks/go/examples/quickstart.go
package main

import (
	"context"
	"fmt"
	"log"
	"os"

	"github.com/lydian/go-sdk"
)

func main() {
	// Initialize client with API key
	client := lydian.NewClient(lydian.Config{
		APIKey: os.Getenv("LYDIAN_API_KEY"),
	})

	ctx := context.Background()

	// Create a new city
	city, err := client.SmartCities.CreateCity(ctx, lydian.City{
		Name:       "San Francisco",
		Country:    "USA",
		Population: intPtr(873965),
	})
	if err != nil {
		log.Fatal(err)
	}

	fmt.Printf("City created: %s\n", city.ID)
}

func intPtr(i int) *int {
	return &i
}

File: docs/sdks/python/examples/quickstart.py
"""Quickstart example for Lydian SDK"""

import os
from lydian import Lydian

# Initialize client with API key
lydian = Lydian(api_key=os.environ.get("LYDIAN_API_KEY"))

# Create a new city
city = lydian.smart_cities.create_city(
    name="San Francisco",
    country="USA",
    population=873965
)

print(f"City created: {city.id}")
